src/rustup.ts

```diff
@@ -78,9 +78,16 @@ async function hasToolchain(): Promise<boolean> {
 async function tryToInstallToolchain(): Promise<void> {
     startSpinner('RLS', 'Installing toolchain…');
     try {
-        const { stdout, stderr } = await execChildProcess(CONFIGURATION.rustupPath + ' toolchain install ' + CONFIGURATION.channel);
-        console.log(stdout);
-        console.log(stderr);
+        const channel = window.createOutputChannel('Rustup');
```

Comment:
FYI this never gets disposed currently. I do not want to dispose of it here after the shell call is over because that would prevent the user from being able to inspect the channel after the toolchain installation. Instead, I think the channel should be returned to the called and added to the extension's `context.subscriptions` so it gets free'd with the extensions deactivation.

I _believe_ if the user was to initiate the toolchain installation twice (possible?), the channel will get reused if it already exists at `createOutputChannel` call-time. This needs to be verified. If yes, we shouldn't register it's destructor with `context.subsriptions` multiple times (so probably should instantiate the output channel outside of this method and pass it in - or it's `append` delegate only).